### PR TITLE
Link Front And Back

### PR DESCRIPTION
**Pull request by zee-programmer91**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
-# if first time
-# Back-End
+## NB: MAKE SURE YOUR VIRTUAL ENVIRONMENT IS ACTIVE AT ALL TIME BEFORE RUNNING THE BELOW
+
+## First Time Run (Installing dependencies)
+
+### (1) Back-End
+
 pip install django
+pip install djangorestframework
```
Comment (XMosuma) [inline README.md]: This one's add them on the requirements file

File: README.md [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
 
+## Run Front-End
 
-# Front-End
-npm install -g expo-cli
-
-# To run the Front-end
 cd front-end
-expo start
+npm run web
```
Comment (XMosuma) [inline README.md]: No need to specify, the expo start will give option on where you wanna run it
Or just run npm start

File: front-end/components/TestPage.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    }, []);
+
+    return (
+        <View style={styles.container}>
+            <Text style={styles.text}>{message}</Text>
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
```
Comment (XMosuma) [inline front-end/components/TestPage.tsx:46]: Next time the styles shouldn't be in the same file as the template, I know it's an example

**Review by XMosuma — APPROVED** [review]

File: front-end/components/TestPage.tsx (hunk 3 above)
Comment (zee-programmer91) [reply front-end/components/TestPage.tsx:46]: I'll leave that to the front end people

File: README.md (hunk 2 above)
Comment (zee-programmer91) [reply README.md]: Ayt. Removed this change.

File: README.md (hunk 1 above)
Comment (zee-programmer91) [reply README.md]: Moved them to the requirements file

**Pull request merged**